WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2170,9 +2170,11 @@ private GutenbergPropsBuilder getGutenbergPropsBuilder() {
         boolean isUnsupportedBlockEditorEnabled =
                 mSite.isWPCom() || (mIsJetpackSsoEnabled && "gutenberg".equals(mSite.getWebEditor()));
 
+        boolean unsupportedBlockEditorSwitch = !mIsJetpackSsoEnabled && "gutenberg".equals(mSite.getWebEditor());
         return new GutenbergPropsBuilder(
                 enableMentions,
                 isUnsupportedBlockEditorEnabled,
+                unsupportedBlockEditorSwitch,
```

Comment:
`unsupportedBlockEditorSwitch` is `true` if the site is a Jetpack site. Meaning that the site can switch the SSO settings. Just Jetpack sites have this capability.
Naming is not the best, but we are trying to make its usage generic, and not refer to SSO or Jetpack directly on `gutenberg`.